.pre-commit-config.yaml

```diff
@@ -28,6 +28,12 @@ repos:
       hooks:
       - id: flake8
         language_version: python3
+  -   repo: https://github.com/codespell-project/codespell
```

Comment:
Totally open to using some other project. I just chose codespell because that's what [`pandas` is using](https://github.com/pandas-dev/pandas/blob/11462d67cd7b857d943e6ce1e27b29ea581eb1fa/.pre-commit-config.yaml#L22-L27)